### DISCUSSION TITLE
Aura is writing snapshots with -B but not finding any with -Br

### DISCUSSION THREAD

**Discussion by TechGameGeek** [fosskers/aura#914] [General]
Hi,
I just started testing AURA at Archlinux. It seems like AURA is taking snapshots with -B, as there are files written to ~/.cache/aura/snapshots.

If I then for example do a AUR install, or lets say a -Syu and try to show snapshots after that by using -Br parameter AURA says No usable snapshots found.

Is this a user-failure, or is there something other wrong?

Thanks!

**Comment by fosskers** [comment]
If you run a `aura check` it'll tell you in the `Snapshots` section if your snapshots are usable. If you're missing tarballs, they won't be. A snapshot is only considered "usable" if _all_ of your currently installed packages have a tarball somewhere in the caches.

**Reply by TechGameGeek** [reply]
@fosskers thank you very much for your reply. I will double check aura check. ;-)
